include/simgrid/s4u/conditionVariable.hpp

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2006-2015. The SimGrid Team. All rights reserved.          */
+
+/* This program is free software; you can redistribute it and/or modify it
+ * under the terms of the license (GNU LGPL) which comes with this package. */
+
+#ifndef SIMGRID_S4U_COND_VARIABLE_HPP
+#define SIMGRID_S4U_COND_VARIABLE_HPP
+
+#include <simgrid/s4u/mutex.hpp>
+
+
+namespace simgrid {
+namespace s4u {
+
+class Mutex;
+
+XBT_PUBLIC_CLASS CondVariable {
+  
+public:
+CondVariable();
```

Comment:
I'd add the constructor taking a `smx_cond_t` (add adding a reference) in order to be able to create `Mutex` from existing `mutex`.